converters/toZigbee.js

```diff
@@ -4075,6 +4036,20 @@ const converters = {
                 for (const member of entity.members) {
                     if (member.meta.hasOwnProperty('scenes') && member.meta.scenes.hasOwnProperty(metaKey)) {
                         membersState[member.getDevice().ieeeAddr] = addColorMode(member.meta.scenes[metaKey].state);
+
+                        let recalledState = member.meta.scenes[metaKey].state;
+
+                        // add color_mode if saved state does not contain it
+                        if (!recalledState.hasOwnProperty('color_mode')) {
+                            recalledState = addColorMode(recalledState);
+                        }
+
+                        // XXX: meta.state is the groups state, we are leaking group state TO devices!
```

Comment:
@Koenkk with this it works, but see the XXX comment.
There has to be a clean way of doing it. This is the last remaining issue together with the toRGB() issue

Reply:
@sjorge wasn't this the case already before? I don't understand how to `color_sync` causes this.